Platform/CometlakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -441,7 +440,7 @@ def main():
     ap.add_argument('-fusa', dest='fusa', action = "store_true", default = False, help = "Patch IFWI to generate Fusa ifwi")
     args = ap.parse_args()
 
-    stitch_cfg_file = imp.load_source('StitchIfwiConfig', args.config_file)
+    stitch_cfg_file = load_source('StitchIfwiConfig', args.config_file)
     if args.work_dir == '':

```

Comment:
Please double check if load_source is available since StitchIfwi.py doesn't import utility modules.
maybe use SourceFileLoader if it is not available.

Reply:
yes, it is available.  CommonUtility is indirectly imported.